game/README.md

```diff
@@ -1,3 +1,13 @@
+# Personal Notes:
```

Comment:
@lizgarciao do you want to remove the personal notes section? Or add them in a different Notes document?